SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -1958,10 +1958,6 @@ void CSolver::InitiateComms(CGeometry *geometry,
       COUNT_PER_POINT  = nVar*3;
       MPI_TYPE         = COMM_TYPE_DOUBLE;
       break;
-    case SOLUTION_DISPONLY:
```

Comment:
👍 Thanks for removing if not needed. Any other stale ones not being used in the FEA solver?

Reply:
Just that one I think.